### ISSUE TITLE
`runPluginVerifier` checks only last version in array

### ISSUE THREAD

**Issue by AlexPl292** [JetBrains/gradle-intellij-plugin#556]
Setup:
```
runPluginVerifier {
  ideVersions = ["IC-2020.1", "IC-2020.2"]
}
```

I may be wrong, but it looks like the plugin is checked against 2020.2 only. This opinion is based on this logline:
```
2020-10-30T10:52:16 [main] INFO  verification - Task check-plugin parameters:
Scheduled verifications (2):
IdeaVIM:SNAPSHOT against IC-202.7660.26, IdeaVIM:SNAPSHOT against IC-202.7660.26
```

And the fact that "reports" directory contains 202 only:
![image](https://user-images.githubusercontent.com/4203721/97674336-6bc29000-1a9e-11eb-984b-9509e636142f.png)

UPD: Just noticed this issue. This may be the cause: https://github.com/JetBrains/gradle-intellij-plugin/issues/553

**Comment by hsz** [comment]
When I set the same `ideVersions` and run the task I get the following output:

```
> Task :runPluginVerifier
Starting the IntelliJ Plugin Verifier 1.252
2020-10-30T10:28:05 [main] INFO  c.j.p.options.OptionsParser - Delete the verification directory /Users/hsz/Projects/JetBrains/intellij-plugin-template/build/reports/pluginVerifier because it isn't empty
Verification reports directory: /Users/hsz/Projects/JetBrains/intellij-plugin-template/build/reports/pluginVerifier
2020-10-30T10:28:06 [main] INFO  verification - Reading IDE /Users/hsz/.pluginVerifier/ides/IC-2020.1
2020-10-30T10:28:09 [main] INFO  verification - Reading IDE /Users/hsz/.pluginVerifier/ides/IC-2020.2
2020-10-30T10:28:12 [main] INFO  verification - Reading plugin to check from /Users/hsz/Projects/JetBrains/intellij-plugin-template/build/distributions/IntelliJ Platform Plugin Template-0.6.0.zip
2020-10-30T10:28:13 [main] INFO  verification - Task check-plugin parameters:
Scheduled verifications (2):
org.jetbrains.plugins.template:0.6.0 against IC-201.6668.121, org.jetbrains.plugins.template:0.6.0 against IC-202.6397.94

2020-10-30T10:28:13 [main] INFO  verification - Finished 1 of 2 verifications (in 0.2 s): IC-201.6668.121 against org.jetbrains.plugins.template:0.6.0: Compatible
2020-10-30T10:28:13 [main] INFO  verification - Finished 2 of 2 verifications (in 0.2 s): IC-202.6397.94 against org.jetbrains.plugins.template:0.6.0: Compatible
Plugin org.jetbrains.plugins.template:0.6.0 against IC-201.6668.121: Compatible
    Plugin can be loaded/unloaded without IDE restart

Plugin org.jetbrains.plugins.template:0.6.0 against IC-202.6397.94: Compatible
    Plugin can be loaded/unloaded without IDE restart

2020-10-30T10:28:13 [main] INFO  verification - Total time spent downloading plugins and their dependencies: 0 ms
2020-10-30T10:28:13 [main] INFO  verification - Total amount of plugins and dependencies downloaded: 0 B
2020-10-30T10:28:13 [main] INFO  verification - Total amount of space used for plugins and dependencies: 0 B
```

`reports` directory also contains two items:

![image](https://user-images.githubusercontent.com/108333/97688899-22bd0c80-1a9b-11eb-9ed6-9b486476cf0c.png)

**Comment by hsz** [comment]
That's correct, that's the same cause.
TL;DR: major IDEs are not present on CDN, so you can't use them for testing anyway + because of an issue on data.services, it points to the latest available version.

**Comment by hsz** [comment]
The list of the available products that can be used for testing is here: https://jb.gg/intellij-platform-builds-list
